packages/trader/src/Stores/Modules/Trading/Constants/validation-rules.ts

```diff
@@ -0,0 +1,241 @@
+import { localize } from '@deriv/translations';
+import { isHourValid, isMinuteValid, isTimeValid, toMoment } from '@deriv/shared';
+import { isSessionAvailable } from '../Helpers/start-date';
+import { TTradeStore } from 'Types';
+
+type TOptions = {
+    min?: number;
```

Comment:
why not number | string as well?

Reply:
good point👍🏻 reused existing TRuleOptions type from validator.ts :)